kafka_topic_enforcer/src/main/java/com/tesla/data/topic/enforcer/ClusterTopics.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2020. Tesla Motors, Inc. All rights reserved.
+ */
+
+package com.tesla.data.topic.enforcer;
+
+import static tesla.shade.com.google.common.base.Preconditions.checkArgument;
+
+import tesla.shade.com.google.common.collect.Sets;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * A utility class to parse consolidated (multi-cluster) topic configurations. The structure of a consolidated config
+ * is,
+ *
+ * <pre>{@code
```

Comment:
nit: dangling `@code`

Reply:
{@code abc} is equivalent to <code>{@literal abc}</code>

Reply:
TIL https://reflectoring.io/howto-format-code-snippets-in-javadoc/